application/views/admin/survey/prepareEditorScript_view.php

```diff
@@ -49,7 +49,7 @@ function find_popup_editor(fieldname)
         return null;
     }
 
-    function start_popup_editor(fieldname, fieldtext, sid, gid, qid, fieldtype, action)
+    function start_popup_editor(fieldname, url)
```

Comment:
param could be called editorURL ? (a bit more descriptive)

Reply:
Yep, and add some phpdoc